hudi-utilities/src/main/java/org/apache/hudi/utilities/UtilHelpers.java

```diff
@@ -578,6 +580,20 @@ public static HoodieTableMetaClient createMetaClient(
         .build();
   }
 
+  public static void addLockOptions(String basePath, TypedProperties props) {
+    if (!props.containsKey(HoodieLockConfig.LOCK_PROVIDER_CLASS_NAME.key())) {
+      HoodieLockConfig lockConfig = HoodieLockConfig.newBuilder()
+          .withLockProvider(FileSystemBasedLockProvider.class)
+          .withLockWaitTimeInMillis(2000L) // 2s
+          .withFileSystemLockExpire(1) // 1 minute
+          .withClientNumRetries(30)
```

Comment:
Use the default values defined in the `ConfigProperty` in `HoodieLockConfig` instead of hard-coding the values?

Reply:
Not sure, I just keep the config same with Flink to make it more friendly to streaming job.